Formula/xbyak.rb

```diff
@@ -0,0 +1,43 @@
+class Xbyak < Formula
+  desc "C++ JIT assembler for x86 (IA32), x64 (AMD64, x86-64)"
+  homepage "https://github.com/herumi/xbyak"
+  url "https://github.com/herumi/xbyak/archive/refs/tags/v6.69.1.tar.gz"
+  sha256 "40b41df141000688d7d629cc9f4bc313f3adfa6be0c42c520064608ed11a93fc"
+  license "BSD-3-Clause"
+  head "https://github.com/herumi/xbyak.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => [:build, :test]
```

Comment:
Can we write the test without needing CMake?

Reply:
it depends on what we want to test. I prefer to test both cmake files and library itself, which requires cmake.

Reply:
Used usual g++ command, hope now it looks more aligned with other brew formulas.